api/turing/config/config_test.go

```diff
@@ -178,84 +179,72 @@ func TestLoad(t *testing.T) {
 		},
 		"single file": {
 			filepaths: []string{"testdata/config-1.yaml"},
-			want: &Config{
+			want: &config.Config{
 				Port:           9999,
 				AllowedOrigins: []string{"http://foo.com", "http://bar.com"},
-				AuthConfig: &AuthorizationConfig{
+				AuthConfig: &config.AuthorizationConfig{
 					Enabled: true,
 					URL:     "http://example.com",
 				},
-				DbConfig: &DatabaseConfig{
+				DbConfig: &config.DatabaseConfig{
 					Host:             "127.0.0.1",
 					Port:             5432,
 					User:             "dbuser",
 					Password:         "dbpassword",
 					Database:         "turing",
 					MigrationsFolder: "db-migrations/",
 				},
-				DeployConfig: &DeploymentConfig{
+				DeployConfig: &config.DeploymentConfig{
 					EnvironmentType: "dev",
 					Timeout:         5 * time.Minute,
 					DeletionTimeout: 1 * time.Minute,
-					MaxCPU:          Quantity(resource.MustParse("500m")),
-					MaxMemory:       Quantity(resource.MustParse("4000Mi")),
+					MaxCPU:          config.Quantity(resource.MustParse("500m")),
+					MaxMemory:       config.Quantity(resource.MustParse("4000Mi")),
 				},
-				KnativeServiceDefaults: &KnativeServiceDefaults{
+				KnativeServiceDefaults: &config.KnativeServiceDefaults{
 					TargetConcurrency:               2,
 					QueueProxyResourcePercentage:    20,
 					UserContainerLimitRequestFactor: 1.25,
 				},
-				RouterDefaults: &RouterDefaults{
+				RouterDefaults: &config.RouterDefaults{
 					LogLevel: "INFO",
-					FluentdConfig: &FluentdConfig{
+					FluentdConfig: &config.FluentdConfig{
 						Tag:                  "turing-result.log",
 						FlushIntervalSeconds: 60,
 					},
-					Experiment: map[string]interface{}{
```

Comment:
Is there no corresponding `Plugins` config test now?

Reply:
I intend to add more tests to this PR, to cover the `plugins_server` part of `servicebuilder` package. Will address it there too, thanks!